eve/main.yml

```diff
@@ -140,8 +140,8 @@ stages:
           command: |
             set -ex
             mkdir -p $CIRCLE_TEST_REPORTS/unit
-            npm run unit_coverage
-            npm run unit_coverage_legacy_location
+            npm test
```

Comment:
Was this change intentional?

Reply:
Yes, we removed this in 8.0, 8.1 but did not backport it.